src/components/FieldOverviewCard/FieldOverviewCard.tsx

```diff
@@ -0,0 +1,80 @@
+import { Card, makeStyles, SvgIconProps, Typography } from '@material-ui/core';
+import { DateTime } from 'luxon';
+import { FC } from 'react';
+import * as React from 'react';
+import { ButtonLink, CardActionAreaLink } from '../Routing';
+
+const useStyles = makeStyles(({ spacing, palette }) => ({
+  card: {
+    width: '321px',
+  },
+  topArea: {
+    display: 'flex',
+    padding: spacing(3, 4, 3, 4),
+    borderBottom: `0.5px solid ${palette.grey[300]}`,
+  },
+  icon: {
+    display: 'flex',
+    alignItems: 'center',
+    justifyContent: 'center',
+    width: '64px',
+    height: '64px',
+    backgroundColor: palette.grey[200],
+    borderRadius: '64px',
+  },
+  rightContent: {
+    paddingLeft: spacing(4),
+  },
+  bottomArea: {
+    display: 'flex',
+    padding: spacing(1, 2, 1, 1),
+    alignItems: 'center',
+    justifyContent: 'space-between',
+  },
+}));
+
+export interface FieldOverviewCardProps {
+  title: string;
+  value: string;
+  icon: React.ComponentType<SvgIconProps>;
+  updatedAt: DateTime;
+  to: string;
+  viewLabel: string;
+}
+
+export const FieldOverviewCard: FC<FieldOverviewCardProps> = ({
+  title,
+  value,
+  icon: Icon,
+  updatedAt,
+  to,
+  viewLabel,
+}) => {
+  const classes = useStyles();
+
+  return (
+    <Card className={classes.card}>
+      <CardActionAreaLink to={to} className={classes.topArea}>
+        <div className={classes.icon}>
+          <Icon />
+        </div>
+        <div className={classes.rightContent}>
+          <Typography color="initial" variant="h4">
+            {title}
+          </Typography>
+          <Typography color="initial" variant="h1">
+            {value}
+          </Typography>
+        </div>
+      </CardActionAreaLink>
+      <div className={classes.bottomArea}>
+        <ButtonLink color="primary" to={to}>
+          {viewLabel}
+        </ButtonLink>
+        <Typography color="textSecondary" variant="body2">
+          {updatedAt.toFormat('MM/dd/yyyy')}
```

Comment:
We want to try to avoid `toFormat` and use `toLocaleString` instead. This uses the `Intl` code to handle differences in...internalization lol (like flipping months and days over the pond).
More info: https://moment.github.io/luxon/docs/manual/formatting.html#tolocalestring--strings-for-humans-

Also, prefix with "Updated ".


```suggestion
          Updated {updatedAt.toLocaleString(DateTime.DATE_SHORT)}
```

Reply:
Got it!